src/reducers/cityReducer.js

```diff
@@ -2,22 +2,26 @@ import * as types from './../actions/types';
 
 export default function cityReducer(state = { name: '', address: '', localization: {} }, action) {
```

Comment:
https://github.com/airbnb/javascript#functions--defaults-last
So... should we add INITIAL_ACTION to const doing nothing?